src/noitedit/search.c

```diff
@@ -454,7 +454,7 @@ protected el_action_t
 cv_search(EditLine *el, int dir)
 {
 	char ch;
-	char tmpbuf[EL_BUFSIZ];
+	char tmpbuf[EL_BUFSIZ-3];
```

Comment:
Any particular reason for dropping the size here?

Reply:
to make sure it isn't too large to copy into the buffer it targets.